simplipy/system/__init__.py

```diff
@@ -250,7 +287,7 @@ def temperature(self) -> int:
         return self._location_info["system"]["temperature"]
 
     @property  # type: ignore
-    @guard_from_missing_data()
+    @guard_from_missing_data(VERSION_V3)
```

Comment:
Why?

Reply:
Because it was causing an error in my tests otherwise. I was getting a `None` version

Reply:
Oh, no longer getting that now. Weird. Reverting